### PR TITLE
feat: Migrate docs

### PR DESCRIPTION
**Pull request by Thogsit**
This PR migrates all current documentation from the GitHub wiki into the `docs` folder. Also, especially the [player instructions](https://github.com/GothicVRProject/GothicVR/tree/feat/migrate-docs/docs/setup/player.md) to setup a working installation have been updated and some minor changes have been made to other doc pages.

You can find the docs for browsing [here](https://github.com/GothicVRProject/GothicVR/tree/feat/migrate-docs/docs).

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+**Welcome to GothicVR Wiki!**  
+Here you find all the information needed to play or contribute to the game.
```
Comment (JucanAndreiDaniel) [inline docs/README.md:2]:
```suggestion
Here you will find all the information needed to play or contribute to the game.
```

File: docs/setup/developer.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+## phoenix-shared-interface
+> Hint: Latest builds to use are stored at the repository's [GitHub Action page](https://github.com/GothicKit/phoenix-shared-interface/actions)
+
+> Hint: If you build a new version of the shared library and put it into GothicVR, you need to store at least both: .dll and .so (for ARM64). Otherwise changes won't be reflected on all VR devices.
+
+This project is required if you need to change C/C++ interface methods for fetching phoenix data.
+
+**Instructions:**
+1. Check out [phoenix-shared-interface](https://github.com/GothicKit/phoenix-shared-interface)
+1. Check the `README.md` file for build instructions
```
Comment (JucanAndreiDaniel) [inline docs/setup/developer.md:28]:
```suggestion
2. Check the `README.md` file for build instructions
```

File: docs/setup/developer.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+1. If you built a new version of `libphoenix-shared.dll/.so`, put it into the GothicVR folder at `Assets/GothicVR/Dependencies`
```
Comment (JucanAndreiDaniel) [inline docs/setup/developer.md:29]:
```suggestion
3. If you built a new version of `libphoenix-shared.dll/.so`, put it into the GothicVR folder at `Assets/GothicVR/Dependencies`
```

**Review by JucanAndreiDaniel — CHANGES_REQUESTED** [review]
Looks fine, just a bit more refinement

**Review by JaXt0r — CHANGES_REQUESTED** [review]

**Review by Tyroonis — APPROVED** [review]

**Comment by Tyroonis** [comment]
Merged
Adjustments follow in succession issue

**Pull request merged**